Deploy-CAM.ps1

```diff
@@ -2695,23 +2723,17 @@ function Set-RadiusSettings() {
     if ($ignorePrompts -and ($enableRadiusMfa -eq $null)) {
         $enableRadiusMfa = $isRadiusMfaEnabled
     }
-    if ($enableRadiusMfa -eq $null) {
-        $currentRadiusSetting = "disabled"
-        if ($isRadiusMfaEnabled) {
-            $currentRadiusSetting = "enabled"
-        }
-        $changeRadiusSettings = (confirmDialog ([String]::Format("Radius Multi-Factor Authentication is currently {0}. Do you want to change Multi-Factor Authentication settings?", $currentRadiusSetting))) -eq 'y'
-    } else {
-        # Check if any settings are different
-        $changeRadiusSettings = ($enableRadiusMfa -notmatch $isRadiusMfaEnabled) -or `
-                                ($radiusServerHost -and ($radiusServerHost -notmatch $currentRadiusHost)) -or `
-                                ($radiusServerPort -and ($radiusServerPort -notmatch $currentRadiusPort)) -or `
-                                ($radiusSharedSecret)
-    }
 
-    # Update Radius Settings appropriatley
-    if ( $changeRadiusSettings ) {
-        # Load provided Radius Configuration Parameters
+    # Check if any settings are different
+    $changeRadiusSettings = `
+        (($enableRadiusMfa -ne $null)   -and ($enableRadiusMfa -notmatch $isRadiusMfaEnabled)) -or `
+        ($radiusServerHost              -and ($radiusServerHost -notmatch $currentRadiusHost)) -or `
+        ($radiusServerPort              -and ($radiusServerPort -notmatch $currentRadiusPort)) -or `
+        ($radiusSharedSecret)
```

Comment:
So we could just compare this easily with a .SecretValueText suffix?

Reply:
We do have the Password in the keyvault as a string, but we'd have to decode the one that a user would pass in. It's doable in a few lines of code to decode it, but I'm not sure if that's something we want to do.

Reply:
yeah okay let's not bother. I agree.